plugins/woocommerce/tests/e2e-pw/envs/default-pressable/playwright.config.js

```diff
@@ -3,12 +3,11 @@ const { devices } = require( '@playwright/test' );
 
 config = {
 	...config,
-	retries: 0,
```

Comment:
Remove this, and [use global PW config](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/tests/e2e-pw/playwright.config.js#L67) - to retry once on CI

Reply:
👍🏼 Makes sense to not override the global config, I think this one was a left over from when I was trying some things out and forgot to remove it.